mxabierto/adela
# Del field byte size

## Changelog:

#### * Modify: In the system Adela was deleted a field callled "byte_size" because, it will be taken from a process called "refineria".

* M:  app/controllers/distributions_controller.rb
* M:  app/serializers/distribution_serializer.rb
* M:  app/views/distributions/edit.html.haml
* M:  config/locales/tooltips/forms/distribution.yml
* M:  spec/features/catalog/distribution_metadata_spec.rb
* M:  spec/requests/data_catalog_management_spec.rb
* M:  spec/requests/data_inventory_management_spec.rb

## TODO:
*  Start up your local environment.
* Execute  rake db:migrate
* Execute rake db:migrate:status : you must see the next migration in state up:
  Remove byte size from distributions

### How to test:
1) Enter to the system Adela and in the option "Plan de Apertura" and add new dataset                       
 and add new distribution and save them.
2) In the option "Catalogo de Datos" you must documented the resource previously created en step 1 and the field "Tamaño en Bytes"  does not appears and save it.
